include/fmt/args.h

```diff
@@ -143,6 +143,38 @@ class dynamic_format_arg_store
   }
 
  public:
+  /**
+    \rst
+    Default constructor of `dynamic_format_arg_store`
+    \endrst
+  */
+  constexpr FMT_INLINE dynamic_format_arg_store() = default;
+
+  /**
+    \rst
+    Constructs a `dynamic_format_arg_store` object from
+    `dynamic_format_arg_store`.
+
+    **Example**::
+
+      fmt::dynamic_format_arg_store<fmt::format_context> store;
+      store.push_back(123);
+      store.push_back("test");
+
+      fmt::dynamic_format_arg_store<fmt::format_context> store2(store);
+    \endrst
+  */
```

Comment:
Let's remove this too because we don't need to explain what a copy constructor is.